startup/csx1/plans/dark_ff.py

```diff
@@ -156,14 +171,14 @@ def _ct_dark(detectors, gain_bit_input, gain_bit_dict):
 
 
 def _ct_dark_cleanup(oldnumim, gain_bit_dict, gain_state,
-                     dark_sh_dict, dark_shutter_state,acq_time, exitslit_pid_init_state ):
+                     dark_sh_dict, dark_shutter_state,acq_time,):# exitslit_pid_init_state ):#DIAG6-out remove line Aug 2024
     print('\nReturning to intial conditions (pre-count).')
     yield from bps.abs_set(fccd.cam.num_images, oldnumim, wait=True)
 
     yield from bps.mv(fccd.cam.fcric_gain, gain_state)
     yield from bps.mv(inout, dark_sh_dict.get(dark_shutter_state))
-    if exitslit_pid_init_state == 1:
-        yield from bps.mv(diag6_pid.enable, 1)
+    #if exitslit_pid_init_state == 1: #DIAG6-out remove line Aug 2024
+    #    yield from bps.mv(diag6_pid.enable, 1) #DIAG6-out remove line Aug 2024
```

Comment:
Small note on tidiness. 
In an ideal setting, we wouldn't leave many commented lines as a way to remember old deletions (this is what git is for). But if it's useful to remember, especially if diag6_pid may come back in the future, then it could be worth placing a single comment in the docstring of the function `_ct_dark_cleanup` that explains the removal made in Aug 2024.

Reply:
sounds good.  the plan is to delete the entire line in August 2024.  Claudio wanted one cycle with it working.